### DISCUSSION TITLE
How to get my Token ?

### DISCUSSION THREAD

**Discussion by BeingTraveler** [community/community#101819] [New to GitHub]
i am need help . PAT personal token ! cant find in dashboard etc help me thanks .

**Comment by GaganHonor** [comment]
Sure 😃
Just click on this link : https://github.com/settings/tokens
and you will see something like 
![image](https://github.com/community/community/assets/141204441/a3a88122-3a0a-44d6-bef0-e882315f4281)
and follow simple steps , thats all 

you can ask more if still having issue

**Comment by BeingTraveler** [comment]
Thanks
